### PR TITLE
Fixup templates

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs [hunk 1]
```diff
@@ -9,11 +9,12 @@ namespace GrpcService_CSharp
 {
     public class GreeterService : Greeter.GreeterBase
     {
-        private readonly ILogger<GreeterService> logger;
-        public GreeterService(ILogger<GreeterService> _logger)
+        private readonly ILogger<GreeterService> _logger;
+        public GreeterService(ILogger<GreeterService> logger)
         {
-            logger = _logger;
+            _logger = logger;
```
Comment (danroth27) [inline src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Services/GreeterService.cs]: Personally I prefer `this.logger = logger`

**Review by danroth27 — CHANGES_REQUESTED** [review]
LGTM. Can we also update the Blazor client templates to align with these changes?

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/WeatherForecast.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Company.WebApplication1
```
Comment (SteveSandersonMS) [inline src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/WeatherForecast.cs]: The namespace doesn’t look consistent with other files in this project. Also why does this need to be added - wasn’t this class already there before?

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs [hunk 3]
```diff
@@ -5,26 +5,35 @@
...
     {
-        private static string[] Summaries = new[]
+        private static readonly string[] Summaries = new[]
         {
             "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
         };
 
-        [HttpGet("[action]")]
-        public IEnumerable<WeatherForecast> WeatherForecasts()
+        private readonly ILogger<WeatherForecastController> _logger;
```
Comment (SteveSandersonMS) [inline src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: Why inject the logger if it’s not used?

Comment (pranavkm) [reply src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: Great Question @SteveSandersonMS. Some questions are best left unanswered.

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/WeatherForecast.cs (hunk 2 above)
Comment (pranavkm) [reply src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/WeatherForecast.cs]: Ahh right. It's in the shared project. The odd namespace meant this file wasn't causing type collisions and causing the build to fail. I'll remove it.

**Comment by pranavkm** [comment]
🆙 📅

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs [hunk 4]
```diff
@@ -5,26 +5,35 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace BlazorHosted_CSharp.Server.Controllers
+namespace BlazorHosted_CSharp.Server.Controller
```
Comment (danroth27) [inline src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: Controller? Isn't this usually plural?

**Review by danroth27 — APPROVED** [review]
One minor nit - but otherwise looks good

**Review by mkArtakMSFT — APPROVED** [review]

File: src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs (hunk 4 above)
Comment (pranavkm) [reply src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/BlazorHosted-CSharp.Server/Controllers/WeatherForecastController.cs]: Yup, that's accidental

**Review by ryanbrandenburg — APPROVED** [review]
Seem good.

**Comment by ElderJames** [comment]
Why are you remove the "api/" prefix at Controller Route Attribute?

**Pull request merged**